src/theme/shared/borderRadius.ts

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
do we need to disable the rule? It seems that all the keys are named fine?

Reply:
True! I will remove it. Thanks 🧼